src/dataExplorer/components/DataExplorerPage.tsx

```diff
@@ -51,6 +51,11 @@ const DataExplorerPageHeader: FC = () => {
   const {scriptQueryBuilder, setScriptQueryBuilder} =
     useContext(AppSettingContext)
   const {resource, save} = useContext(PersistanceContext)
+  const isNewIOxOrg = useSelector(selectIsNewIOxOrg)
+  const showDataExplorerToggle =
```

Comment:
Out of context, it might be easy to think `showDataExplorerToggle` is a method that shows the Data Explorer toggle - it reads like a verb, which is why I typically recommend a conjugated verb like `should` a la `shouldShowDataExplorerToggle`.

Reply:
good point, I'll update the rest of variables names in this files to follow the same naming convention